packages/docs/tests/EndToEnd/Docs.browser.spec.ts

```
@@ -1,36 +1,33 @@
+import type {
+    ParentSidebarItem,
+    SidebarItem,
+}                             from '@/.vitepress/theme/Components/Layout/Features/Sidebar/Types/Sidebar';
+import type { Locator }       from 'playwright';
```

ESLint

[error] Expected "playwright" (type) to come before "@/.vitepress/theme/Components/Layout/Features/Sidebar/Types/Sidebar" (internal-type).

(perfectionist/sort-imports)